cache/subtopic.go

```diff
@@ -0,0 +1,56 @@
+package cache
+
+import (
+	"strings"
+	"sync"
+)
+
+// SubtopicsIDs contains a list of subtopics in map form with addition to mutex locking
+// The subtopicsMap is used to keep a record of subtopics to be later used to generate the subtopics id `query` for a topic
+// and to check if the subtopic id given by a user exists
+type SubtopicsIDs struct {
+	mutex        *sync.RWMutex
+	subtopicsMap map[string]bool
```

Comment:
So we will need the subtopic name/label and release date as well as the topic ids; these need to be stored in the cache for use by the census hub page view/template

Reply:
Root topic (census) shouldn't be included